TheMovieDb/Reviews/View/ReviewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ReviewCell.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 31/10/21.
+//
+
+import UIKit
+
+final class ReviewCell: UICollectionViewCell {
+    
+    // MARK: - Properties
+    static let reuseIdentifier =  String(describing: ReviewCell.self)
+
+    public var review: Review? {
+        didSet {
+            configure()
+        }
+    }
+    private let logoImageView = BackgroundImageView()
+    
+    private let authorNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.boldSystemFont(ofSize: 18)
+        label.numberOfLines = 0
+        label.adjustsFontSizeToFitWidth = true
+        return label
+    }()
+    
+    private let descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.boldSystemFont(ofSize: 16)
+        label.numberOfLines = 0
+        return label
+    }()
+    // MARK: - Life Cycle
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureUI()
+    }
+    
+    required init?(coder: NSCoder) {
```

Comment:
@available(*, unavailable) 

this is a suggestion, you can add this piece of code at the top of the function and make a method not available so nothing can use it (it's ideal to avoid someone else uses it in the wrong way) :)

Reply:
Added